src/TransactionTestsFiller/ttEIP1559/maxFeePerGas00prefixFiller.json

```diff
@@ -0,0 +1,24 @@
+{
+    "maxFeePerGas00prefix" :
+    {
+	    "expectException" :
+        {
+          "<=Berlin" : "TR_TypeNotSupported",
+          ">=London" : "GasLimitPriceProductOverflow"
+        },
+        "transaction" :
+        {
+            "data" : "",
+            "gasLimit" : "21000",
+            "maxFeePerGas" : "0x00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
```

Comment:
Same. 00 prefix

Reply:
Updated to have "0x:bigint". Fails as expected with exception "1559LeadingZerosBaseFee". Separated max gas value test in new file.